sahi/postprocess/legacy/combine.py

```diff
@@ -0,0 +1,213 @@
+# OBSS SAHI Tool
+# Code written by Fatih C Akyon, 2021.
+
+import copy
+from typing import List, Union
+
+import numpy as np
+
+from sahi.annotation import BoundingBox, Category, Mask
+from sahi.prediction import ObjectPrediction
+
+
+def calculate_area(box: Union[List[int], np.ndarray]) -> float:
+    """
+    Args:
+        box (List[int]): [x1, y1, x2, y2]
+    """
+    return (box[2] - box[0]) * (box[3] - box[1])
+
+
+def calculate_intersection_area(box1: np.ndarray, box2: np.ndarray) -> float:
+    """
+    Args:
+        box1 (np.ndarray): np.array([x1, y1, x2, y2])
+        box2 (np.ndarray): np.array([x1, y1, x2, y2])
+    """
+    left_top = np.maximum(box1[:2], box2[:2])
+    right_bottom = np.minimum(box1[2:], box2[2:])
+    width_height = (right_bottom - left_top).clip(min=0)
+    return width_height[0] * width_height[1]
```

Comment:
I think we can change here like this for better readability.

```python
width, height = (right_bottom - left_top).clip(min=0)
return width * height
````

Reply:
One more thing, these helper functions also exist on `sahi/postprocess/combine.py`, to avoid duplication we can move these to some util script. Wdyt ? @fcakyon

Reply:
that would make sense 👍